androidApp/src/main/java/co/nimblehq/avishek/phong/kmmic/android/ui/screen/surveydetail/SurveyDetailScreen.kt

```diff
@@ -0,0 +1,47 @@
+package co.nimblehq.avishek.phong.kmmic.android.ui.screen.surveydetail
+
+import androidx.compose.foundation.pager.*
+import androidx.compose.material.*
+import androidx.compose.runtime.*
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.tooling.preview.PreviewParameter
+import androidx.lifecycle.compose.collectAsStateWithLifecycle
+import co.nimblehq.avishek.phong.kmmic.android.ui.theme.ApplicationTheme
+import co.nimblehq.avishek.phong.kmmic.presentation.module.HomeViewModel
+import org.koin.androidx.compose.getViewModel
+
+@Composable
+fun SurveyDetailScreen(
+    homeViewModel: HomeViewModel,
+    surveyId: String,
+    onBackClick: () -> Unit
+) {
+    val viewState = homeViewModel.viewState.collectAsStateWithLifecycle()
+    val surveyUiModel = viewState.value.surveys.find { it.id == surveyId }
```

Comment:
```suggestion
    val viewState by homeViewModel.viewState.collectAsStateWithLifecycle()
    val surveyUiModel = viewState.surveys.find { it.id == surveyId }
```

Reply:
Updated in https://github.com/nimblehq/avishek-phong-kmm-ic/pull/118/commits/1a902df4fd37f70ff2d375c7321a16fc067846ee